app/views/posts/index.html.erb

```diff
@@ -1,6 +1,8 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Posts</h1>
+<p><%= link_to('Generate Report CSV', posts_path(format: 'csv')) %></p>
```

Comment:
Parenthesis for the `link_to` method is unnecessary, rails know that every sentence that follows are arguments.